### ISSUE TITLE
All1+Attr

### ISSUE THREAD

**Issue by veramal** [tuffnatty/suddenly#194]
туразархы - нет разбора
Возможно, мы это запрещали сознательно, но не могу найти, где это. Лог разбора что-то тоже на дает внятной информации в последнее время.
В общем, надо бы это уметь разбирать и сделать правило, которое бы выключали (опционально) гармонию для этого сочетания (т.е. при переднем ряде тоже чтобы было зархы, а не зеркi).

**Comment by veramal** [comment]
Вытаскиваю из письма: 
Обнаружила, что мы исключили сочетание _зархы_ (All САр + Attr КЫ) из словоизменения за то, что оно не подчиняется сингармонизму. Т.е. все такие случаи решили пустить через словарь? Однако сочетание с _орты_ дает разнообразие форм, и все их записывать в словарь глупо: _**Ортызархызынаң** чӧріп одыр!_ -  иди по тому, которое находится в середине!, 
_Павел, **ортызынзархы** чирлерӌе иртіп_ - Павел, пройдя верхние страны, 
_Кимнің **ортынзархызында** паза ӱстӱнзаринда чуртааннар_ - населяли верховья Енисея до среднего течения.
Есть пара примеров со словами, которые не описывают положение в пространстве: _сугзархы, изiксархы_.

**Comment by adybo** [comment]
Ну вставляем, у нас же уже много падежей без гармонии

**Comment by veramal** [comment]
Падежей без гармонии у нас не много (времен много). Но главное не это, а то, что я не могу найти, какое правило запрещает это сочетание.

**Comment by tuffnatty** [comment]
Это сочетание запрещено гармонией. *_туразарғы_ разбирается.

**Comment by veramal** [comment]
Тогда тут возможны два решения: писать дополнительный вариант аффикса _кЫ_ и правило сочетаемости либо считать _Сархы_ целиком одним аффиксом. Второй вариант привлекает тем, что можно и гармонию убрать сразу (с вариантом _серкi/зеркi_ 1 пример всего). Аня?

**Comment by adybo** [comment]
Там же не только зархы, в словаре вон еще
 
аархы дальний  алтынзархы северный  андархы находящийся в тех краях  ибіркі близлежащий  индіркі нижний  іскеркі восточный  кидеркі западный  мындархы местный  озархы находящийся на той стороне  оңархы правый  ӧӧркі верхний
пееркі ближайший  тасхархы  наружный  тӧбінкі нижний  тоғырхы поперечный  тӧрзеркі передний угол  ӱстӱнзархы южный  хайдархы находящийся в какой-либо стороне  чоғархы верхний
т.е. это от местоименного направительного на -Ар
"Локативные" же производные без озвончения почти все образованы либо от окаменевшей формы аллатива на  -САР(Ы) , либо от формы на  -ГАр , которая, по О.В.Субраковой [БХРС: 1075, приложение], оформляет "наречные образования", исторически же является падежным показателем директива [Erdal 2004: 177]. Единственное исключение -  тӧбінкі 'нижний', которое, как мы видим, еще у Вербицкого давало озвончение. 
 
В общем, по совокупности, наверное, надо делать САркы, хотя не понимаю, чем обозначать к/х, это не К
мы должны вводить для различения формы этих аффиксов новую морфонему: это К 1 , озвончающееся в интервокальной позиции и не озвончающееся после сонанта (ср. атрибутивы от названий места типа  аал-да-ғы  'сельский' ), в отличие от ранее введенного К, озвончающегося и в интервокальной позиции, и после сонанта, которое фигурирует в аффиксе, сочетающемся с временными названиями ( Г - другая морфонема, которая в интервокальной позции выпадает; в привременном атрибутивизаторе  фигурирует не она, ср.  киӌее-гі 'вчерашний'. )

**Comment by veramal** [comment]
Сархы, как я уже упоминала, не гармонирует - есть только 1 пример с гармонией при массе употреблений _ӱстӱнзархы_. 
То, что в словаре, вводить в парсер не предлагаю, т.к. это непродуктивные сочетания. Второй вариант был оставить сочетание All1 САр + Attr, но добавить для Attr вариант хы (строчными буквами).

**Comment by adybo** [comment]
Тогда и в Сар надо а маленькое тут писать

**Comment by veramal** [comment]
Ввела в таблицу All1.Attr Сархы в слот Transp (13).  Дописала про него условие в правило 16.1, написала отдельное ограничение 16.9, чтобы не цеплял падеж в первой позиции (хотя, наверное, и без ограничения не будет цеплять).

**Comment by adybo** [comment]
ура

**Comment by veramal** [comment]
В копилку: бывают образования с -гы и от чистой глагольной основы.
Хырғы пычах салба - Не поражай (нас) ножом истребления (Катанов-Радлов,
III.109)
Хырғы < хыр 'истреблять, уничтожать'

сб, 8 мая 2021 г. в 18:45, adybo ***@***.***>:

**Comment by adybo** [comment]
Да, но это не атрибутивизатор, а одно из множества номин акционисов

**Comment by veramal** [comment]
А вот это надо сделать.

**Comment by tuffnatty** [comment]
В табличке (н)Сархы, в остальных упоминаниях Сархы.
Сделал как в табличке, тогда разбирается только туранзархы, естественно. Переделал без (н), - наверное, в табличке опечатка.

**Comment by adybo** [comment]
Вера, погляди. По-моему, норм

**Comment by veramal** [comment]
Не норм. Н - посессивное склонение. Должно разбираться и с ним, и без него:
туразархы, туранзархы.

**Comment by tuffnatty** [comment]
Тогда надо заводить отдельные посессивный и непосессивный варианты, как это происходит с Case1/Case2. И правило их выбора.

**Comment by veramal** [comment]
А можно завести два варианта этого аффикса в парсере: Сархы обычный и (н)Сархы посессивный? А в самой табличке пусть останется один, там и так (н) в скобках. Правило такое же, как у Case1/Case2: (н)Сархы выбирается после аффиксов 1pos.sg1, 2pos.sg1, 3pos1, в прочих случаях выбирается Сархы.

**Comment by veramal** [comment]
Но можно проще сделать: нСархы после 3pos1, в прочих случаях Сархы. Тогда в
парсере 2 варианта без скобок, а в описании 1 со скобками.

**Comment by tuffnatty** [comment]
А где в туранзархы 3pos1?

**Comment by veramal** [comment]
Это была неправильная форма, правильная - туразынзархы.

**Comment by tuffnatty** [comment]
Сделал так: All₁.Attr Сархы / All₁.Attr (н)Сархы. Выбор осуществляется по правилу 17, как в <Case₁>. Неплохо бы привести табличку и правила в соответствие с парсером.

**Comment by veramal** [comment]
Дописала аффикс в правило 17, вроде больше ничего не нужно.
